charts/shoot-core/components/charts/apiserver-proxy/templates/apiserver-proxy-daemonset.yaml

```diff
@@ -0,0 +1,120 @@
+---
+apiVersion: {{ include "daemonsetversion" . }}
+kind: DaemonSet
+metadata:
+  name: apiserver-proxy
+  namespace: kube-system
+  labels:
+    garden.sapcloud.io/role: system-component
+    origin: gardener
+spec:
+  updateStrategy:
+    type: RollingUpdate
+  selector:
+    matchLabels:
+      app: kubernetes
+      role: apiserver-proxy
+  template:
+    metadata:
+      annotations:
+        scheduler.alpha.kubernetes.io/critical-pod: ''
+        checksum/proxy-config: {{ include ( print $.Template.BasePath "/proxy-configmap.yaml") . | sha256sum }}
+      labels:
+        garden.sapcloud.io/role: system-component
+        origin: gardener
+        app: kubernetes
+        role: apiserver-proxy
+        networking.gardener.cloud/to-dns: allowed
+        networking.gardener.cloud/to-apiserver: allowed
+    spec:
+      serviceAccountName: apiserver-proxy
+      priorityClassName: system-cluster-critical
+      tolerations:
+      - effect: NoSchedule
+        operator: Exists
+      - key: CriticalAddonsOnly
+        operator: Exists
+      - effect: NoExecute
+        operator: Exists
+      hostNetwork: true
+      automountServiceAccountToken: false
+      initContainers:
+      - name: setup
+        image: {{ index .Values.images "apiserver-proxy-sidecar" }}
+        imagePullPolicy: IfNotPresent
+        args:
+        - --ip-address={{ .Values.advertiseIPAddress }}
+        - --setup-iptables=false
+        - --daemon=false
+        - --interface=lo
+        securityContext:
+          capabilities:
+            add:
+            - NET_ADMIN
+        resources:
+          requests:
+            cpu: 20m
+            memory: 20Mi
+          limits:
+            cpu: 60m
+            memory: 40Mi
+        volumeMounts:
+        - name: systembussocket
+          mountPath: /var/run/dbus/system_bus_socket
```

Comment:
Why is this needed?

Reply:
IIRC the iptables was not working without it. I'll test it tomorrow.